database/factories/RoomFactory.php

```diff
@@ -29,16 +29,16 @@ public function definition()
             'D',
         ];
         $price = [
```

Comment:
let's keep this change on cloned repo only i think? since overall of this app built with rupiah